validate/validate.go

```diff
@@ -85,3 +87,58 @@ func RegularCFilter(block *wire.MsgBlock, filter *gcs.Filter) error {
 	}
 	return nil
 }
+
+// CFilterV2HeaderCommitment ensures the given v2 committed filter has actually
+// been committed to in the header, assuming dcp0005 was activated.
+func CFilterV2HeaderCommitment(net wire.CurrencyNet, header *wire.BlockHeader, filter *gcs2.FilterV2, leafIndex uint32, proof []chainhash.Hash) error {
+	const opf = "validate.CFilterV2HeaderCommitment(%v)"
+
+	// FIXME: fill with actual values
+	const (
+		cfilterv2RawHashes_mainnet = ""
```

Comment:
fine for now until it activates but we'll want to write a generator for creating this code. would be better to keep the data as binary too instead of hex encoded.

Reply:
Might also be a good idea to export it from an internal package, that way the db code can use it as well instead of having to fetch these over the net.

Reply:
Oh, i see they were generated for testnet.  And that's a massive file. Hmmmm

Reply:
As discussed in the chat: 

> i think it should be a manual validation for now, add an rpc that hashes all of the unvalidated filters and checks it against the hardcoded commitment
> and maybe also an rpc to repair from trusted data

Reply:
For added completeness, I suggested this after seeing the bloat to the dcrwallet executable, and this is before adding the mainnet filters:

```
% goweight | head
   21 MB github.com/decred/dcrwallet/validate
  5.4 MB github.com/decred/dcrwallet/rpc/walletrpc
  4.0 MB net/http
  3.6 MB runtime
  3.5 MB github.com/decred/dcrwallet/wallet/v3
  2.5 MB github.com/decred/dcrwallet/wallet/v3/udb
  2.1 MB golang.org/x/net/http2
  2.0 MB github.com/golang/protobuf/proto
  1.9 MB net
  1.8 MB crypto/tls
% go build
% ls -lah ./dcrwallet
-rwxr-xr-x  1 jrick  staff    43M Feb 11 12:38 ./dcrwallet
```